addons/mail/static/src/models/chatter.js

```diff
@@ -280,11 +280,16 @@ registerModel({
                     },
                 });
                 this.createNewRecordDeferred.resolve();
-                this.update({
-                    createNewRecordComposerData: clear(),
-                    createNewRecordDeferred: clear(),
-                });
             }
+            if (this.createNewRecordFiles) {
```

Comment:
Possibly all of those restore (even those that existed before) should only be done on the main "if" where thread is is set above?

Doing it here on top level might explain why sometimes it is done when the thread is not ready yet?

Reply:
Should not be moved inside the `if` or it doesn't work at all.
I get the recovering doesn't pass in `this.threadId` conditional